### DISCUSSION TITLE
Can I have a recurring formula run every X hours?

### DISCUSSION THREAD

**Discussion by timfong888** [AntlerVC/firetable#1009] [Support / Q&A] [answered]
In my application, different users at different times are assigned 100 life points when a user record is created. 

I would like life points to decrement randomly 1-5 points per 1-3 days from the day the record was created and recur on that interval. 

Can I do something like this with Rowy?

**Comment by BeeBombshell** [comment, marked as answer]
Hey @timfong888, this use case is a bit more challenging. You would require a cloud function that runs on a cron job to see whether sufficient time has passed to decrement points.
There is no way to create these types of functions using Rowy at the moment. However, we're planning to include that as well as some of the more generic functions. Hopefully, it will be available in the later releases of Rowy, stay tuned! 🚀

**Reply by harinij** [reply]
Hi @timfong888 we have an upcoming release for this, stay tuned :) You can follow our roadmap now here: https://roadmap.rowy.io/
